flathub.json

```
@@ -0,0 +1,3 @@
+{
+    "skip-arches": ["aarch64"]
```

this in disappointing given that godot support aarch64.

```suggestion
    "only-arches": ["x86_64"]
```

---

the BaseApp support aarch64 too.